s3-connector/build.gradle.kts

```diff
@@ -0,0 +1,292 @@
+        exclude(group = "log4j")
+        exclude(group = "org.apache.commons", module = "commons-text")
+        exclude(group = "org.slf4j", module = "slf4j-api")
+        exclude(group = "org.apache.hadoop", module = "hadoop-auth")
+        exclude(group = "org.apache.hadoop", module = "hadoop-yarn-api")
+        exclude(group = "com.google.re2j")
+        exclude(group = "com.google.protobuf")
+        exclude(group = "com.google.code.gson")
+        exclude(group = "com.jcraft")
+        exclude(group = "org.apache.curator")
+        exclude(group = "org.apache.zookeeper")
+        exclude(group = "org.apache.htrace")
+        exclude(group = "com.google.code.findbugs")
+        exclude(group = "org.apache.kerby")
+        exclude(group = "com.fasterxml.jackson.core")
+        exclude(group = "com.fasterxml.woodstox", module = "woodstox-core:5.0.3")
+        exclude(group = "org.apache.avro", module = "avro")
+        exclude(group = "org.apache.hadoop", module = "hadoop-yarn-common")
+        exclude(group = "com.google.inject.extensions", module = "guice-servlet")
+        exclude(group = "io.netty", module = "netty")
+    }
+
+    // Make test utils from 'test' available in 'integration-test'
+    integrationTestImplementation(sourceSets["test"].output)
+    integrationTestImplementation("org.awaitility:awaitility:4.2.1")
+}
+
+tasks.named<Pmd>("pmdIntegrationTest") {
+    ruleSetFiles = files("${project.rootDir}/gradle-config/aiven-pmd-test-ruleset.xml")
+    ruleSets = emptyList() // Clear the default rulesets
+}
+
+tasks.named<SpotBugsTask>("spotbugsIntegrationTest") {
+    reports.create("html") {
+        setStylesheet("fancy-hist.xsl")
+    }
+}
+
+tasks.processResources {
+    filesMatching("s3-connector-for-apache-kafka-version.properties") {
+        expand(mapOf("version" to version))
+    }
+}
+
+tasks.jar {
+    manifest {
+        attributes(mapOf("Version" to project.version))
+    }
+}
+
+tasks.distTar {
+    dependsOn(":commons:jar")
+}
+tasks.distZip {
+    dependsOn(":commons:jar")
+}
+
+distributions {
+    main {
+        contents {
+            from("jar")
+            from(configurations.runtimeClasspath.get().map { if (it.isDirectory) it else zipTree(it) })
+
+
+            into("/") {
+                from("$projectDir")
+                include("version.txt", "README*", "LICENSE*", "NOTICE*", "licenses/")
+                include("config/")
+            }
+        }
+    }
+}
+
+publishing {
+    publications {
+        create<MavenPublication>("publishMavenJavaArtifact") {
+            groupId = group.toString()
+            artifactId = "s3-connector-for-apache-kafka"
+            version = version.toString()
+
+            from(components["java"])
+
+            pom {
+                name = "Aiven's S3 Sink Connector for Apache Kafka"
+                description = "Aiven's S3 Sink Connector for Apache Kafka"
+                url = "https://github.com/aiven-open/s3-connector-for-apache-kafka"
```

Comment:
Do we keep these urls still ?

Reply:
For now, subsequent PR for fixing required.